docs/source/diff-quality.rst

```diff
@@ -0,0 +1,60 @@
+.. _diff-quality:
+
+For projects with large amounts of (potentially imperfect) SQL code, the full
+SQLFluff output could be very large, which can be distracting -- perhaps the CI
+build for a one-line SQL change shouldn't encourage the developer to fix lots
+of unrelated quality issues.
+
+To support this use case, SQLFluff integrates with a quality checking tool
+called `diff-quality`. By running SQLFluff using `diff-quality` (rather than
+running it directly), you can limit the the output to the new or modified SQL
+in the branch (aka pull request or PR) containing the proposed changes.
+
+Currently, ``diff-quality`` requires that you are using ``git`` for version
+control.
+
+Installation
```

Comment:
@barrywhart - given that `diff-cover` is now in the `requirements.txt` for `sqlfluff`, users shouldn't need to install it seperately. Can you update this section to explain that and say that it's packaged as part of sqlfluff?

Otherwise this looks great.

Reply:
Oh -- great point!! Maybe this whole section is unnecessary. Will review.